common/environment.js

```diff
@@ -1,2 +1,7 @@
 //NOTE(amine): feature flags
 export const ACTIVITY_FEATURE_FLAG = !!process.env.NEXT_PUBLIC_ACTIVITY_FEATURE_FLAG;
```

Comment:
Can you update this to use NEXT_PUBLIC_NODE_ENV instead? And update the code that checks for it to check if NEXT_PUBLIC_NODE_ENV === "staging". The possible values for it will be development, staging, and production. It's already been added to the staging and production servers